src/repr/src/explain_new/mod.rs

```diff
@@ -657,17 +664,29 @@ where
     let mut first_rows = Vec::with_capacity(20);
     for _ in 0..20 {
         if let Some((row, diff)) = rows.next() {
-            row_count += diff;
+            row_count += diff.abs();
             first_rows.push((row, diff));
         }
     }
-    let rest_of_row_count = rows.into_iter().map(|(_, diff)| diff).sum::<crate::Diff>();
-    if rest_of_row_count != 0 {
-        writeln!(f, "{}total_rows: {}", ctx, row_count + rest_of_row_count)?;
-        writeln!(f, "{}first_rows:", ctx)?;
-        ctx.indented(move |ctx| write_first_rows(f, &first_rows, ctx))?;
+    if row_count > 0 {
+        let rest_of_row_count = rows
+            .into_iter()
+            .map(|(_, diff)| diff.abs())
+            .sum::<crate::Diff>();
+        if rest_of_row_count != 0 {
+            writeln!(
+                f,
+                "{}total_rows (diffs absed): {}",
+                ctx,
+                row_count + rest_of_row_count
+            )?;
+            writeln!(f, "{}first_rows:", ctx)?;
+            ctx.indented(move |ctx| write_first_rows(f, &first_rows, ctx))?;
+        } else {
+            write_first_rows(f, &first_rows, ctx)?;
+        }
     } else {
-        write_first_rows(f, &first_rows, ctx)?;
+        writeln!(f, "{}<empty>", ctx)?;
```

Comment:
* Can we use `Constant empty`, `Constant ∅`, or `Const total_rows=0 abs_diffs=0` here?
* Can you make this change in other parts that render `Constant